product_variant_cost_price/__init__.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Ainara Galdona - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from . import models
+from openerp import SUPERUSER_ID
+
+
+def load_cost_price_on_variant(cr, registry):
+    product_obj = registry['product.product']
```

Comment:
To be able to configure 'cost_method' there had to be configured in Warehouse configuration, by default it is disabled.
So I tested... I have enabled it and then look into the db looking for the cost_method, but only if I change one of them it creates the ir_property